moxin-frontend/src/landing/model_files_item.rs

```diff
@@ -195,18 +215,33 @@ impl Widget for ModelFilesItem {
 
 impl WidgetMatchEvent for ModelFilesItem {
     fn handle_actions(&mut self, cx: &mut Cx, actions: &Actions, scope: &mut Scope) {
-        if let Some(fd) = self.view(id!(download_button)).finger_down(&actions) {
-            if fd.tap_count == 1 {
-                let widget_uid = self.widget_uid();
-                let Some(model) = &self.model else { return };
-                let Some(file) = &self.file else { return };
-
-                cx.widget_action(
-                    widget_uid,
-                    &scope.path,
-                    ModelFileItemAction::Download(file.clone(), model.clone()),
-                );
-            }
+        let widget_uid = self.widget_uid();
+
+        if self.cbutton(id!(download_button)).tapped(&actions) {
```

Comment:
Yeah, I didn't want to imply a "mouse device" when this may be eventually be used in mobile so for me "tapped" or "[pressed](https://api.flutter.dev/flutter/material/ButtonStyleButton/onPressed.html)" are more "cross-platform" terms, but I agree that we should make everything consistent for now 👍

Reply:
Change done